src/client/components/Dropdown/DropDown.js

```diff
@@ -24,7 +24,7 @@ DropDown.propTypes = {
   setValue: PropTypes.func,
   items: PropTypes.arrayOf(
     shape({
-      id: PropTypes.string,
+      id: PropTypes.number,
```

Comment:
Are you sure this change didn't break existing usages of `Dropdown` like in storybook or elsewhere?

Reply:
I had a look over all the components in storybook,  it doesn't seen anything like breaking. In database id is number for all the tables, so this is the reason I have changed PropType to number